TempMail/utils/Email.swift

```diff
@@ -10,26 +10,27 @@
 import SwiftUI
 
 class Email : ObservableObject{
-    @Published var email_addr:String?
+    @Published var email_addr: String
     
-     func GenEmail() {
+//  init method
+    init() {
+        self.email_addr = ""
+        genEmailAddr()
+    }
+//  generates an email address
+    func genEmailAddr() -> Void {
         //list of acceptable chars
-         let letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
-         //returns a random permutation of letters; string of up to length 9 for email addr
-         self.email_addr = String((0..<9).map{ _ in letters.randomElement()!})
-     }
-     
-     //getter for email addr
-     func getEmailAddr() -> String {
-         if email_addr != nil{
-             return "Current Address: \(email_addr!)@1secmail.com"
-         }
-         else{
-             return "Press \"Generate New Email\" to get started! "
-         }
-     }
-    //getter for emails
-    func getEmails(){
-        //make the HTTP GET request to return the JSON shit
+        let letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+//      returns a random permutation of letters; string of up to length 9       for email addr
+        self.email_addr = String((0..<9).map{ _ in letters.randomElement()!})
+    }
+    
+//  getter for email addr
+    func getEmailAddr() -> String {
+        return "Current Address: \(email_addr)@1secmail.com"
```

Comment:
I think we should let the ContentView handle the string formatting (Current Address: ) and instead just return email_addr. Gives better separation of view and model.